### DISCUSSION TITLE
Is there a way to send panel to specific monitor?

### DISCUSSION THREAD

**Discussion by v4u6h4n** [ErikReider/SwayNotificationCenter#374] [Q&A]
Hey everyone :-)

I'm just wondering if there is a way to send to the panel to a specific monitor? I've done quite a bit of searching through docs and can't find references to this anywhere in github issues. At the moment I'm using waybar to launch with `swaync-client -t -sw` from the waybar example config. I'm running Hyprland if that's relevant, and there aren't any available rules for setting monitors or workspaces for layers.

**Comment by Aligehri** [comment]
Maybe exec a script that exec swaync and then moveit to a specific workspace. You can set a workspace in a monitor with workspaces rules

use this:
"hyprctl dispatch movetoworkspace number"

**Comment by everyonesaythankyouted** [comment]
OMFG. I finally found it and I even logged into GitHub so that my descendants could find it from the first link in Google.

`swaync-client --change-noti-monitor DP-3`

It resets when you reboot the config. You know what to do next.
